project_timeline/__manifest__.py

```diff
@@ -5,7 +5,7 @@
 {
     "name": "Project timeline",
     "summary": "Timeline view for projects",
-    "version": "11.0.1.1.0",
+    "version": "11.1.0.0.0",
```

Comment:
@InfernalLV `11.0.1.0.0`

Reply:
It should `11.0.1.2.0`

Reply:
I see what I got wrong. But shouldn't it then be `11.0.2.0.0`? This pull request addresses a breaking change in OCA/web#1206. Doesn't that mean that this pull request introduces a breaking change as a consequence of the first one? Which Is why I went for the bump on the highest digit.

Reply:
Yes, 11.0.2.0.0 makes sense.